pages/docs/community/styleguide/accessibility.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Accessibility
+description: This style guide explains how to include accessibility in the documentation.
+weight: 20
+---
+
+## Accessibility
+
+At AsyncAPI, we strive to make our documentation/content inclusive, accessible, and unbiased to everyone. We encourage all contributors to have diversity and inclusivity in mind when writing. To ensure this, we have provided an overview of general guidelines.
+
+### Language
+- Be clear and concise when writing. Avoid the use of complex language or technical jargon. For example, instead of using "*MQTT Server Binding*," it could be simplified to "*Protocol-specific information for an MQTT server*," or "*WebSockets Operation Binding*" can be explained as "*Protocol-specific information for WebSockets operation*."
+- Keep paragraphs and sentences short, simple, and to the point.
+- Always maintain a uniform structure. Use descriptive headings and subheadings to make navigation easy.
+- Use inclusive language and always keep the reader in mind when writing. 
+
+
+### Text
+- Use the appropriate heading hierarchy. H1 is used for the main heading while H2 to H6 are used for subsection headings. 
+- Properly align text for easy readability.
+- Avoid using camel case or any unnecessary fonts and formatting.
+- Define acronyms or abbreviations upon the first use in your documentation, and always spell out any signs or symbols.
+- Structure your text in a uniform format by maintaining consistent structure throughout your docs. Accurately provide clear and concise descriptions of AsyncAPI concepts and elements as referenced throughout the documentation. 
+
+### Links
+- Use descriptive and meaningful link text. For example, do not use phrases like *click here* or *follow this link* instead use *see*.
```

Comment:
```suggestion
- Use descriptive and meaningful link text. For example, do not use phrases like *click here* or *follow this link* instead use *see*.
```

See..what? To make it descriptive, link text must describe **where the link takes them**.

Please write a new example 😄

Reply:
🤣🤣 you are schooling me on accessibility, and I'm glad I'm learning

Reply:
♥️🔥🔥🔥 it's definitely a mindset change